src/maintainer/adding_pkgs.rst

```diff
@@ -363,9 +363,16 @@ while adding ``pip`` to the host requirements:
       host:
         - pip
 
-These options should be used to ensure a clean installation of the package without its
-dependencies. This helps make sure that we're only including this package,
-and not accidentally bringing any dependencies along into the conda package.
+`conda-forge` configures `pip` to ensure a clean installation of the package
```

Comment:
This is actually conda build, not conda-forge. A link to what conda-build sets would be useful. I have not been able to find one.

Reply:
In gitter `jakirkham` linked https://github.com/conda/conda-build/blob/061063f10c51162707c7c5253daa39b2752fcfcc/conda_build/build.py#L3169-L3189

I'll add that reference to the note here. May have to circle back on this though as pre-xmas is getting hectic...

Reply:
I changed this to `conda-build` now but feel free to add the note too!